tests/integration/seed/seedsVerification.test.js

```diff
@@ -206,6 +203,17 @@ describe('SEEDS VERIFICATION', () => {
           expect(someInterAttendanceSheetHasDate).toBeFalsy();
         });
 
+        it('should pass if only inter courses have trainee in attendance sheet', () => {
+          const someTraineesDontExist = attendanceSheetList.some(a => a.course.type === INTER_B2B && !a.trainee);
+
+          expect(someTraineesDontExist).toBeFalsy();
+
+          const someIntraAttendanceSheetHasTrainee = attendanceSheetList
+            .some(a => a.course.type === INTRA && a.trainee);
+
+          expect(someIntraAttendanceSheetHasTrainee).toBeFalsy();
+        });
```

Comment:
Que penses-tu d'ajouter un test pour verifier que si on est en inter on a bien le champs trainee ?

Reply:
je vérifie déjà ça ligne 207 dans le test should pass if only inter courses have trainee in attendance sheet